ui/src/app/pages/access/access-renewal.tsx

```diff
@@ -131,7 +131,11 @@ const syncAndReloadTraining = fp.flow(
 });
 
 // Helper Functions
+// The module has already expired
+export const hasExpired = (expiration: number): boolean =>
+  getWholeDaysFromNow(expiration) < 0;
 
+// The module can either be expired or is expiring
 export const isExpiring = (expiration?: number): boolean =>
```

Comment:
This function does not differentiate between expired and expiring, hence had to create a new method for expired

Reply:
sgtm